### PR TITLE
Fix the Request action dialogs for the requests' list

### PR DESCRIPTION
**Pull request by lgalis**
Fix the Request action dialogs for the requests' list.

### CODE REVIEW

File: src/smart-components/request/requests.js [hunk 1]
```diff
@@ -29,13 +29,13 @@ const Requests = ({ fetchRequests, requests, pagination, history }) => {
   const routes = () => <Fragment>
     <Route exact path="/requests/add_comment/:id" render={ props => <ActionModal { ...props }
       actionType={ 'Add Comment' }
-      preMethod = { fetchRequest }
+      fetchData = { true }
```
Comment (Hyperkid123) [inline src/smart-components/request/requests.js]: There is a convention If you want to pass `true` boolean value as a prop. You can should just use its name:
```
<Component ... fetchData />
```

File: src/smart-components/request/action-modal.js [hunk 2]
```diff
@@ -1,29 +1,31 @@
...
   closeUrl,
-  preMethod,
-  postMethod,
-  requestId
+  postMethod
 }) => {
+  const [ selectedRequest, setSelectedRequest ] = useState({});
+
+  const fetchData = (setRequestData) => {
+    fetchRequest(id).payload.then((data) => { setRequestData(data);});
```
Comment (Hyperkid123) [inline src/smart-components/request/action-modal.js]: no need fo brackets around single line functions

```javascript
const fetchData = setRequestData =>
  fetchRequest(id).payload.then(data => setRequestData(data));
```

File: src/smart-components/request/requests.js (hunk 1 above)
Comment (lgalis) [reply src/smart-components/request/requests.js]: Thanks for pointing this out. I ended up not using this prop in the component, so I will remove it as a prop here, but I'll use the convention next time it's needed.

File: src/smart-components/request/action-modal.js (hunk 2 above)
Comment (lgalis) [reply src/smart-components/request/action-modal.js]: Sounds good. ( had a debug statement that I removed and did not also remove the brackets).

**Comment by lgalis** [comment]
@Hyperkid123 - feedback addressed.

**Pull request merged**